folding/base/neuron.py

```diff
@@ -155,8 +155,8 @@ def sync(self):
         if self.should_sync_metagraph():
             self.resync_metagraph()
 
-        if self.should_set_weights():
-            self.set_weights()
+        # if self.should_set_weights():
```

Comment:
We need to move weight setting away from sync so that we don't set weights whenever we call sync in the `def run` function